LIFECYCLE.md

```diff
@@ -0,0 +1,16 @@
+# Lifecycle Management
+
+## Device Lifecycle scope
+
+High level requirements on a DPU/IPU or subsequently a "device"
+
+* a way to reboot a device
+  * if connected to a host, the host should be able to survive a device reset
+* a way to update a device
+  * firmware update
+  * OS update
+  * software/application update
+* a way to recover a device
+  * reset to a known good state, e.g., factory reset
+* a way to debug a device
```

Comment:
cover aspects to find out if the device crashed, how to debug, recover, and provide post-mortem information
how to debug problems, etc.
More importantly - is there a desire to do this uniformly across various xPUs?

Reply:
at least fetch logs or core dumps, for example are desired to be uniformed... maybe more...

Reply:
Yes, I think there is a desire to uniformly collect failure data and recover.  Perhaps the format of that data is different, and of course the specifics of live debugging a card will be different for different hardware and software vendors.